torch/csrc/jit/ivalue.h

```diff
@@ -75,10 +75,31 @@ struct Shared {
   PointerType * pImpl;
 };
 
+// string
+struct ConstantString : at::Retainable {
+ private:
+  ConstantString(const std::string & str)
+  : str_(str) {}
+  const std::string & str_;
```

Comment:
This should be `const std::string`. There is no guarantee that  `str` will stay valid after this function call.